h2o-docs/src/product/data-science/glm.rst

```diff
@@ -101,7 +101,9 @@ Defining a GLM Model
 
 -  `standardize <algo-params/standardize.html>`__: Specify whether to standardize the numeric columns to have a mean of zero and unit variance. Standardization is highly recommended; if you do not use standardization, the results can include components that are dominated by variables that appear to have larger variances relative to other attributes as a matter of scale, rather than true contribution. This option is enabled by default.
 
--  `missing_values_handling <algo-params/missing_values_handling.html>`__: Specify how to handle missing values (Skip or MeanImputation).
+-  `missing_values_handling <algo-params/missing_values_handling.html>`__: Specify how to handle missing values (Skip, MeanImputation, or PlugValues).
+
+-  **plug_values**: When ``missing_values_handling=plug_values``, specify a single row frame containing values that will be used to impute missing values of the training/validation frame.
```

Comment:
whoops! I made a mistake. This should be:
"... When ``missing_values_handling="PlugValues"``, specify ..."